charts/rancher-backup/templates/deployment.yaml

```diff
@@ -65,4 +72,4 @@ spec:
       tolerations: {{ include "linux-node-tolerations" . | nindent 8 }}
 {{- if .Values.tolerations }}
 {{ toYaml .Values.tolerations | indent 8 }}
-{{- end }}
+{{- end }}
```

Comment:
We probably need to decide wether we want newlines at end of file or not, this is usually a code editor setting and if we don't decide what we do, we keep getting these changes over and over. I've seen more PRs with a newline than without.